redbot/core/bot.py

```diff
@@ -1240,10 +1247,19 @@ async def get_command_setting(guild_id: int) -> Optional[bool]:
             scope = self._config.custom(COMMAND_SCOPE, command.qualified_name, guild_id)
             return await scope.embeds()
 
+        if isinstance(channel, discord.GroupChannel) or isinstance(channel, discord.DMChannel):
+            # we should not be passing group channels to this function
+            # this only uses global settings
```

Comment:
I'm not entirely sure this comment makes that much sense in here as it did when it had a different place to be in so it might make sense to remove it, your call.